CheckDrive.Api/CheckDrive.Infrastructure/Extensions/DependencyInjection.cs

```diff
@@ -1,5 +1,5 @@
-﻿using CheckDrive.Domain.Interfaces;
-using CheckDrive.Infrastructure.Configurations;
+﻿using CheckDrive.Domain.Authorization;
```

Comment:
in which place of this file this namespace is used?

Reply:
Add Jwt Options, I am replaced it to Domain from Infrastructure, because i could not use it in application layer

Reply:
you don't need it, use UserManager instead